utils/keys.go

```diff
@@ -95,15 +95,15 @@ func ExportKeys(to io.Writer, s Exporter, from string) error {
 func ImportKeys(from io.Reader, to []Importer, fallbackRole string, fallbackGUN string, passRet notary.PassRetriever) error {
 	// importLogic.md contains a small flowchart I made to clear up my understand while writing the cases in this function
 	// it is very rough, but it may help while reading this piece of code
-	data, err := ioutil.ReadAll(from)
+	byteData, err := ioutil.ReadAll(from)
```

Comment:
given we already had the import alias, this is an unnecessary variable name change.